languages/go/exercises/concept/maps/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+Further Reading:
+
+- [godoc-race-detector]
+- [goblog-race-detector]
+- [goblog-iteration-order]
+
+A `map` in Go is an associative data type that contains a collection of key/value pairs, which other languages might refer to as as dictionary or an associative array.
+
+The value of an uninitialized map is `nil`, and you can define a map as follows:
+
+```go
+make(map[string]int)
+```
+
+or
+
+```go
+  var foo map[string]int
+  foo = make(map[string]int)
+```
```

Comment:
I don't see the difference between the two ways to create a map, they both use the same function. Is there a difference that I am missing?

Is the other way to define a map:

```go
map[string]int{}
```

?

Reply:
> Is the other way to define a map:
Yep that's my point. My argument is that `make` can be used to initialize some built in data type like slice, map, and channel. I believe it's worth to mention you can also define `map` with `make`. I also leave out that you can specify the initial capacity when defining a map.

Reply:
> Yep that's my point.

But I don't see `map[string]int{}` listed as an example? 😕

Reply:
Sorry about that and the previous comment 🤦‍,
I fixed it in 6229bd5

Reply:
Seems you only fixed it in `introductions.md`...